.github/workflows/continuous-integration.yml

```diff
@@ -87,7 +87,7 @@ jobs:
           envsubst < .config/.env.example > .env
           python manage.py makemigrations
           python manage.py migrate
-          coverage run manage.py test
+          coverage run manage.py test --keepdb
```

Comment:
I wonder if this is necessary though? Because the CI rebuilds every time.

Reply:
Yeah, I think you're right. Removed in 72acae31ee82cac8f81401eb6ed927911e619134.